docs/build/tutorials/platform/subnets/create-a-evm-blockchain-on-subnet-with-avalanchejs.md

```diff
@@ -0,0 +1,545 @@
+# Deploying Subnet with EVM Based Blockchain using AvalancheJS
+
+## Introduction
+
+Blockchain is an instance of **Virtual Machine (VM)** that processes transactions to change the genesis (base) and the subsequent states. VMs define blockchain’s state, state transition function, transactions, and the API through which users can interact with the blockchain. On Avalanche, VM serves as the blueprint for creating different instances of independent blockchains, yet sharing the same ruleset.
+
+All transactions on a blockchain need to be validated by the distributed nodes or validators. **Subnets** are the group of validators that can validate one or many chains. On Avalanche, there is one primary subnet, that currently validates 3 blockchains, and each chain serves a different purpose. One can easily deploy their subnets and invite other validators to be a part of it. A validator can be a part of multiple subnets, but it must be validating the primary subnet.
+
+On Avalanche, out of the 3 chains, **Platform Chain** (P-Chain) manages subnets, subnet validators, subnet blockchain, etc. In this tutorial, you will learn about creating your subnet and deploying **Ethereum Virtual Machine (EVM)** based blockchain on that subnet through your Node.js application using **AvalancheJS**. AvalancheJS is a javascript library that allows you to issue commands to the Avalanche node APIs without worrying about transaction serialization, signing it with the keys etc. On our [subnet series of tutorials](https://docs.avax.network/build/tutorials/platform/subnets/), you can get various great articles explaining subnets, building custom VMs, and a lot more.
+
+
+## Requirements
+
+* [Go](https://golang.org/doc/install) version >= 1.16.8
+* [NodeJS](https://nodejs.org/en) Node >= 10.16 and npm >= 5.6
+
+## Project structure
+
+Open the terminal and make a new directory at your desired location. We will keep all the binaries and other codes in this folder.
+
+```bash
+mkdir subnet-evm-demo
+cd subnet-evm-demo
+```
+
+## Setting up AvalancheGo and Subnet EVM Binaries
+
+Clients interact with Avalanche blockchain by issuing API calls to the nodes running `AvalancheGo`. We have to clone the repository, build its binary, and run it. If we want to deploy our blockchain, we have to put the blockchain's VM binary inside the `build/plugins` directory. Here we will also clone the `subnet-evm` repository, build the VM's binary, and copy it to the AvalancheGo's `build/plugins` directory. Follow the steps below -
+
+### Clone AvalancheGo Repository
+
+```bash
+git clone https://github.com/ava-labs/avalanchego
+cd avalanchego
+```
+
+### Build Binary
+
+Running the below command will create an `avalanchego` binary inside the `build/` directory and will install the coreth `evm` binary inside the `build/plugins` directory.
+
+```bash
+./scripts/build.sh
+```
+
```

Comment:
Please add something similar, replacing the direct URL to a relative URL.

`For more information on running a node, please refer [this](https://docs.avax.network/build/tutorials/nodes-and-staking/run-avalanche-node).`